language/google/cloud/language/entity.py

```diff
@@ -176,4 +184,7 @@ def from_api_repr(cls, payload):
         metadata = payload['metadata']
         salience = payload['salience']
         mentions = [Mention.from_api_repr(val) for val in payload['mentions']]
-        return cls(name, entity_type, metadata, salience, mentions)
+        sentiment = None
+        if payload.get('sentiment'):
```

Comment:
Should be `if payload.get('sentiment') is not None` but you probably just want `if 'sentiment' in payload`